### DISCUSSION TITLE
DWD MOSMIX Zeitstempel und Gültigkeit der Daten.

### DISCUSSION THREAD

**Discussion by hoetzgit** [roe-dl/weewx-DWD#37] [Q&A]
Ich habe eine Frage zu den DWD MOSMIX Daten. In der Datensatzbeschreibung des DWD für die MOSMIX Daten ist mir aufgefallen, dass oft Formulierungen wie z.B. "... während der letzten Stunde ..." auftauchen.

Nehmen wir das Beispiel `FX1` mit Beschreibung "Maximale Windböe **innerhalb der letzten Stunde**" und einen Zeitpunkt **12:17** für die Abfrage.

Wenn ich also um **12:17** wissen möchte, wie die Bedingungen lt. MOXMIX Modell für diesen Zeitpunkt bestimmt wurden, muss ich dann den Datensatz **12:00** oder **13:00** verwenden? Wenn ich die Beschreibung richtig interpretiere, dann ist um **13:00** `FX1` für die letzte Stunde, also **12:00** bis **13:00** enthalten. Dann müsse ich also den Datensatz **13:00** nehmen wenn ich **12:17** wissen möchte? Oder sind im Datensatz **12:00** die Daten für **12:00** bis **13:00** enthalten?

**Comment by roe-dl** [comment]
Ich interpretiere es so, daß der Datensatz für 13:00 Uhr den Zeitraum von 12:00 Uhr bis 13:00 Uhr umfaßt.
